tests/utils/test_run_dbt_cloud_job.py

```diff
@@ -0,0 +1,131 @@
+import json
+from pathlib import Path
+
+import pytest
+from responses import Response
+
+from utils.run_dbt_cloud_job import trigger_dbt_run, poll_dbt_run
+
+
+def test_should_trigger_run(responses):
+    # GIVEN: DBT cloud ready to accept job
+    rsp = Response(method="POST", url="https://cloud.getdbt.com/api/v2/accounts/1001/jobs/101/run/",
```

Comment:
@ifoukarakis , can we store this url in a static var of some sort - it's being reused everywhere.